felix.go

```diff
@@ -466,6 +466,13 @@ func servePrometheusMetrics(configParams *config.Config) {
 				prometheus.Unregister(prometheus.NewProcessCollector(os.Getpid(), ""))
 			}
 		}
+		gaugeHost := prometheus.NewGauge(prometheus.GaugeOpts{
+			Name:        "felix_host",
+			Help:        "Configured Felix hostname (as a label), typically used in grouping/aggregating stats; the label defaults to the hostname of the host but can be overridden by configuration. The value of the gauge is always set to 1.",
+			ConstLabels: prometheus.Labels{"host": configParams.FelixHostname},
+		})
+		prometheus.MustRegister(gaugeHost)
```

Comment:
Just spotted this was inside the loop.  Does it need to be?  Is registering idempotent?

The `configParams` can't change BTW, Felix restarts if they do.